cliTool/index.ts

```diff
@@ -585,6 +587,23 @@ async function app(): Promise<any> {
         await Sudo.batchAsSudoFinalized(tx);
         console.log(message.toString());
       }
+      if (
+        answers.option.includes(
+          "Activate and claim 3rd party rewards to default users",
+        )
+      ) {
+        await inquirer
+          .prompt([
+            {
+              type: "input",
+              name: "user",
+              message: "default //Charlie",
+            },
+          ])
+          .then(async (answers: { user: string }) => {
+            activateAndClaim3rdPartyRewardsForUser(answers.user);
```

Comment:
plz default it as charlie as you mentioned above,

Reply:
Fixed